modules/common/src/main/java/org/opencastproject/job/api/AbstractJobProducer.java

```diff
@@ -173,22 +173,32 @@ public boolean isReadyToAccept(Job job) throws ServiceRegistryException, Undispa
     // Note: We are not adding the job load in the next line because it is already accounted for in the load values we
     // get back from the service registry.
     float currentLoad = getServiceRegistry().getOwnLoad();
+    logger.debug("{} Current load on this host: {}, job's load: {}, job's status: {}, max load: {}", new Object[] {
```

Comment:
Don't use an array here. Just remove the `new Object[] { }` and it will still work.